torch/_dynamo/optimizations/torchxla_integration.py

```diff
@@ -166,12 +163,12 @@ def optimized_mod(*args):
         # need to convert xla tensor back to eager tensor
```

Comment:
update this comment?  you don't need to convert xla to eager anymore.  but you still have a timer for inplace copy_ for syncing any input mutations back to input tensors right?

Reply:
Good catch.  The timer for 'copy result' should be removed together with the comments. For 'inplace updates' yes, we still have a timer for that.